README.md

```diff
@@ -46,6 +46,8 @@ Note: If you are using root (not recommended on your dev machine, maybe in a con
 
 This project uses [Yarn workspaces](https://yarnpkg.com/blog/2017/08/02/introducing-workspaces/).
 
+In order to package or build a release version of the Manager, you will need [jq](https://stedolan.github.io/jq/) to parse `package.json`
```

Comment:
Let's use [jq-node](https://www.npmjs.com/package/node-jq) as a dev dependency so it gets installed with yarn.

Reply:
Done in 5a647d9abf6896c01e85dbb8f1343f522bf2fee6